src/components/WidgetText/WidgetTextPresentation.js

```diff
@@ -97,7 +97,7 @@ function WidgetTextPresentation({ widgetData }) {
             <div className="rounded-full shadow">
               <a
                 href={url}
-                {...(url.includes(FIRSTVOICESLINK)
+                {...(url.includes(FIRSTVOICESLINK) || widgetData.sitename
```

Comment:
If I guessed right about the goal here, I could imagine a lot of scenarios where an external url would include the site slug. It would probably be better to check whether the url starts with either of the two values, rather than just containing one of them